packages/store-indexer/bin/postgres-indexer.ts

```diff
@@ -66,6 +66,14 @@ async function getLatestStoredBlockNumber(): Promise<bigint | undefined> {
   }
 }
 
+async function getDistanceFromFollowBlock(): Promise<bigint> {
+  const [latestStoredBlockNumber, latestFollowBlockNumber] = await Promise.all([
+    (await getLatestStoredBlockNumber()) ?? -1n,
+    (await publicClient.getBlock({ blockTag: env.FOLLOW_BLOCK_TAG })).number,
+  ]);
+  return latestFollowBlockNumber - latestStoredBlockNumber;
```

Comment:
this doesn't do anything because the `await` is in line with array, so you're calling `Promise.all` on already-awaited values
```suggestion
  const [latestStoredBlockNumber, latestFollowBlockNumber] = await Promise.all([
    getLatestStoredBlockNumber().then((blockNumber) => blockNumber ?? -1n),
    publicClient.getBlock({ blockTag: env.FOLLOW_BLOCK_TAG })).then((block) => block.number),
  ]);
  return latestFollowBlockNumber - latestStoredBlockNumber;
```

Reply:
alternatively
```suggestion
  const [latestStoredBlockNumber, latestFollowBlock] = await Promise.all([
    getLatestStoredBlockNumber(),
    publicClient.getBlock({ blockTag: env.FOLLOW_BLOCK_TAG }),
  ]);
  return (latestFollowBlockNumber ?? -1n) - latestStoredBlock.number;
```

Reply:
haha damn i even remembered an article about this common mistake while writing this